internal/unifi/client.go

```diff
@@ -25,14 +25,19 @@ type httpClient struct {
 }
 
 const (
-	unifiLoginPath  = "%s/api/auth/login"
-	unifiRecordPath = "%s/proxy/network/v2/api/site/%s/static-dns/%s"
+	unifiLoginPathUDM         = "%s/api/auth/login"
+	unifiLoginPathNetworkApp  = "%s/api/login"
+	unifiRecordPathUDM        = "%s/proxy/network/v2/api/site/%s/static-dns/%s"
+	unifiRecordPathNetworkApp = "%s/v2/api/site/%s/static-dns/%s"
 )
 
 // newUnifiClient creates a new DNS provider client and logs in to store cookies.
 func newUnifiClient(config *Config) (*httpClient, error) {
+	log.Debug("Creating new UniFi client", zap.String("host", config.Host), zap.String("controllerType", config.ControllerType))
+
 	jar, err := cookiejar.New(&cookiejar.Options{PublicSuffixList: publicsuffix.List})
 	if err != nil {
+		log.Error("Failed to create cookie jar", zap.Error(err))
```

Comment:
This error is superfluous since it's already bubbled up on the call of `newUnifiClient` here

https://github.com/kashalls/external-dns-unifi-webhook/blob/5e8a4911f1b8d8751dd869f4a62d41950167c549/internal/unifi/provider.go#L24-L26